doc/configurations/nudged_simulations.rst

```diff
@@ -16,6 +16,16 @@ use your own compset and add::
   
 to CAM_CONFIG_OPTS in env_build.xml after creating a case.
 
+IMPORTANT NOTE (2022.09.06): 
```

Comment:
```suggestion
IMPORTANT NOTE (2022.09.06): 
""""""""""""""""""""""""""""
```
Should this be a paragraph header? RST does not respect newline characters.

Reply:
I support this suggestion. Makes it more visible for the readers.